src/command/list/human.rs

```diff
@@ -41,14 +50,55 @@ pub(super) fn format(toolchain: &Toolchain) -> Option<String> {
 /// Accepts the components *from* the toolchain rather than the item itself so
 /// that
 fn display_active(
-    node: &Option<Node>,
-    package_manager: &Option<PackageManager>,
+    runtime: &Option<Box<Node>>,
+    package_manager: &Option<Box<PackageManager>>,
     packages: &[Package],
 ) -> String {
-    match (node, package_manager, packages) {
-        (None, _, _) => NO_RUNTIME.to_string(),
-        (Some(node), Some(package_manager), packages) => unimplemented!(),
-        _ => unimplemented!(),
+    match (runtime, package_manager) {
+        (None, _) => NO_RUNTIME.to_string(),
+        (Some(runtime), Some(package_manager)) => {
+            let runtime_version: String =
+                WRAPPER.fill(&format!("Node: {}", format_runtime(runtime)));
+            let package_manager_version: String = WRAPPER.fill(&format!(
```

Comment:
Minor nit: `Wrapper::fill` returns a `String`, so the type annotations aren't necessary.